### ISSUE TITLE
warning while using sort 

### ISSUE THREAD

**Issue by fireballpoint1** [plumed/plumed2#534]
```
PLUMED:   added component to this action:  sort.787
PLUMED: WARNING for action SORT with label sort : a description of component 788 has not been added to the manual. Components should be registered like keywords in registerKeywords as described in the developer doc.
```
What is to be done to get rid of the warning and what are its possible repercussions of ignoring the same?
I am doing something like the following in my plumed file currently: 
```
d0: DISTANCES GROUPA=2 GROUPB=1,5,9 MIN={BETA=0.1}
d1: DISTANCES GROUPA=3 GROUPB=1,5,9 MIN={BETA=0.1}
d2: DISTANCES GROUPA=4 GROUPB=1,5,9 MIN={BETA=0.1}
sort: SORT ARG=d0.min,d1.min,d2.min
restraint-all: RESTRAINT ARG=sort.3 KAPPA=5 AT=1.3
PRINT STRIDE=1 ARG=sort.1,sort.789,restraint-all.bias FILE=COLVAR
```

**Comment by GiovanniBussi** [comment]
@carlocamilloni should we solve this for 2.6 as well?

**Comment by carlocamilloni** [comment]
Yes I missed it
It should be easy and suggests that it is not regtested

Sent from my iPhone

**Comment by carlocamilloni** [comment]
Actually I fixed this in July from v2.5 on
@fireballpoint1 anyway ignoring this warning does not cause any problem
